src/plugins/auto/tests/unit/parse_meta_device_test.cpp

```diff
@@ -41,8 +41,8 @@ const char igpuFullDeviceName[] = "Intel(R) Gen9 HD Graphics (iGPU)";
 const char dgpuFullDeviceName[] = "Intel(R) Iris(R) Xe MAX Graphics (dGPU)";
 // const char myriadFullDeviceName[] = "Intel Movidius Myriad X VPU";
```

Comment:
Please update device name in comments from VPUX to NPU

Reply:
Done.